### PR TITLE
Move framework examples higher and mention popular CMSes

### PR DESCRIPTION
**Pull request by fureigh**
This PR adds links to our Drupal, WordPress and SuiteCRM examples so those terms will be indexed for site search and otherwise made easier to find on the [Languages and Frameworks](https://cloud.gov/docs/apps/frameworks/) page of the docs.

Context: potential customers have been asking whether they can run specific CMSes on cloud.gov. Though several are represented in our `cf-example-*` repos, they're currently discoverable only via the`cf-hello-worlds` repo's README.

Minimally closes https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-6009.

### CODE REVIEW

**Comment by brittag** [comment]
Are the Wordpress and SuiteCRM examples functional / reasonably up to date? If they're not, we probably shouldn't direct people to them, since they would be faced with a potentially frustrating experience...

File: content/docs/apps/frameworks.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ cloud.gov uses [buildpacks]({{< relref "docs/getting-started/concepts.md#buildpa
 
 cloud.gov supports applications written in Go, Java, Node.js, .NET Core, PHP, Python, and Ruby. cloud.gov also supports applications that rely on a static binary that uses the 64-bit Linux kernel ABI, or that consist of static HTML, CSS, and Javascript assets. See the [Cloud Foundry system (supported) buildpacks list](http://docs.cloudfoundry.org/buildpacks/#system-buildpacks) for details.
 
+## Examples
+
+Check out [cloud.gov's collection of Hello World applications](https://github.com/18F/cf-hello-worlds) and [Cloud Foundry's sample applications](https://github.com/cloudfoundry-samples) for examples in many different languages and frameworks, including [Drupal](https://github.com/18F/cf-ex-drupal), [WordPress](https://github.com/18F/cf-ex-wordpress) and [SuiteCRM](https://github.com/18F/cf-example-suitecrm).
```
Comment (jcscottiii) [inline content/docs/apps/frameworks.md]: I wonder if we should break this down into a list of options. 1) Hello Worlds 2) CloudFoundry sample apps 3) Drupal 4) Wordpress 5) SuiteCRM. (not for or against it, just thinking) any thoughts?

Comment (brittag) [reply content/docs/apps/frameworks.md]: That makes sense to me, to give each option a little more room.

Comment (fureigh) [reply content/docs/apps/frameworks.md]: @jcscottiii I took a shot in https://github.com/18F/cg-site/pull/946/commits/b134078883e4752b6fa897996df9b35934229213. Ready for another look!

**Review by jcscottiii — APPROVED** [review]

**Comment by fureigh** [comment]
@brittag I haven't tried the WordPress and SuiteCRM examples. I included them only because I saw them in our [cf-hello-worlds README](https://github.com/18f/cf-hello-worlds).

**Pull request merged**